templates/egressnetworkpolicy.go

```diff
@@ -33,7 +33,7 @@ var EgressNetworkPolicyTemplate = openshiftv1.EgressNetworkPolicy{
 				To: openshiftv1.EgressNetworkPolicyPeer{
 					CIDRSelector: "0.0.0.0/0",
 				},
-				Type: openshiftv1.EgressNetworkPolicyRuleDeny,
+				Type: openshiftv1.EgressNetworkPolicyRuleAllow,
```

Comment:
We are allowing `pagerduty` events to talk to us, but is that the only thing that will be able to communicate after this change? Not sure what's happening here, can you please elaborate?

Reply:
@rexagod This is not an accepted case, because of the previous code changes we were not able to load the DF console page, this is temp workaround

Reply:
ACK